docs/proposals/emptydir-pv.md

```diff
@@ -0,0 +1,151 @@
+# Emptydir persistent volume
+
+This document proposes a model for managing emptydir persistent volumes backed by node local storage.
+
+## Abstract
+We currently have:
+* Various different types of "local volumes", like emptyDir, hostPath and gitRepo
+* A way to "claim" storage, such that the pod referencing the claim can continue using the storage across pod/node restarts
+* A way to dynamically provision volumes to match un-bound/pending claims
+
+The second and third features allow us to say: provision a volume on demand, that survives pod/node restarts by using network attached storage (NFS, network block device from cloudprovider, distributed file system like gluster). This proposal is about a new `emptyDir` persistent volume that combines all three features. The idea is that we can sacrifice some durability to get a similar abstraction using *node local storage*. We expect this new PV source to be useful to applications that:
+* Prioritize latency over durability (eg: async replicated database using local ssd)
+* Prioritize cost over durability (eg: dev/test prototyping)
+* Bring their own out of band save/restore snapshotting
+
+## Goals
```

Comment:
Is there any plan to support disk quota management for emptydir?  The use case is same as that you mentioned "we can sacrifice some durability to get a similar abstraction using *node local storage*".  It avoids that containers run out of disks. Actually, there are many applications that use a lot of local disk in our cluster, so we need manage disk quota for them.  

Another idea: let containers use more than one local disk instead of just one local dir "/var/.../kubelet/...". Then we could make the best use of all the local disk on kubelet nodes.